.github/workflows/scheduled.yml

```diff
@@ -711,18 +711,42 @@ jobs:
   row-number-fuzzer-run:
     name: RowNumber Fuzzer
     runs-on: ubuntu-latest
-    container: ghcr.io/facebookincubator/velox-dev:centos9
+    container: ghcr.io/facebookincubator/velox-dev:presto-java
```

Comment:
nit: Let's rename line 711 to be `presto-java-row-number-fuzzer-run:` to be clear that it compares with Presto-Java.

Reply:
Done. Thanks.